Learning_FEM_1D_step0.ipynb

```diff
@@ -347,38 +347,41 @@
+    "$$R(u_h):=q-(-\\frac{d}{dx}D\\frac{du_h}{dx}+\\sigma u_h)$$\n",
+    "That is why the inequality.)\n",
     "\n",
     "This is not very useful. First, our approximate solution cannot be differentiated twice in $x$. Second, we need to obtain equations **for all** of the unknown nodal values $u_j$ with $0 \\le j \\le N$.\n",
     "\n",
     "One way to solve these issues is to integrate that equation over the entire domain (getting rid of the $x$ variable)\n",
     "\n",
-    "$$ \\int_0^L dx \\left( -\\frac{d}{dx}D\\frac{du_h}{dx} + \\sigma u_h = q \\right)$$\n",
+    "$$ \\int_0^L dx \\left( -\\frac{d}{dx}D\\frac{du_h}{dx} + \\sigma u_h \\ne q \\right)$$\n",
     "or\n",
     "$$ -\\left.D\\frac{du_h}{dx}\\right|_L + \\left.D\\frac{du_h}{dx}\\right|_0 + \\int_0^L dx \\, \\sigma u_h =  \\int_0^L dx  q $$\n",
     "\n",
-    "The derivatives can be expressed as functions of $u_0, u_1$, and $u_{N-1},u_N$ and the integral on the left-hand-side will depend on all values of $u_i$. This is not very useful, still, because all the unknowns are present in a single equation. We need **more equations!**\n",
+    "(That is we integrate both side of the equality of the original equation and expect that the results are equal for some combination of $u_j$ s.) The derivatives can be expressed as functions of $u_0, u_1$, and $u_{N-1},u_N$ and the integral on the left-hand-side will depend on all values of $u_j$. This is not very useful, still, because all the unknowns are present in a single equation. We need **more equations!**\n",
     "\n",
     "How many equations do we need? We need $N+1$ equations because we have $N$ segments and thus $N+1$ vertices.\n",
     "\n",
     "How many basis functions do we have? We have $N+1$.\n",
     "\n",
-    "There is the answer. We will integrate the governing law over space. But, before doing this, we will multiply our governing law by **any** of those $N+1$ basis functions. **We call this 'multiplying by a test function'**. Let us denote by $b_i(x)$ any of those $N+1$ possible choices. The index $i$ will denotes each equation we will derive. Think of $i$ aas the being the index for rows in a matrix.\n",
+    "There is the answer. We will integrate the governing law over space. But, before doing this, we will multiply our governing law by **any** of those $N+1$ basis functions. **We call this 'multiplying by a test function'**. Let us denote by $b_i(x)$ any of those $N+1$ possible choices. The index $i$ will denotes each equation we will derive. Think of $i$ as the being the index for rows in a matrix.\n",
     "\n",
     "* Step-0: multiply governing law by $b_i(x)$ for any $0 \\le i \\le N$ \n",
     "$$\\phantom{a}$$ \n",
-    "$$ -b_i(x)\\frac{d}{dx}D\\frac{du_h}{dx} + b_i(x)\\sigma u_h = b_i(x)q \\qquad \\forall x \\in [0,L]$$\n",
+    "$$ -b_i(x)\\frac{d}{dx}D\\frac{du_h}{dx} + b_i(x)\\sigma u_h \\ne b_i(x)q \\qquad \\forall x \\in [0,L]$$\n",
```

Comment:
remove \\ne and replace with \quad \text{and} \quad

Reply:
Changed as per suggestion